edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-control-numbers.adoc

```diff
@@ -0,0 +1,28 @@
+= EDIFACT EDI Connector 2.2 - Control Numbers - Mule 4
+
+Using the Write operation in the Anypoint EDIFACT EDI Connector, you can generate control numbers on the generated EDIFACT payload by:
```

Comment:
I'm guessing the full name is Anypoint Connector for EDIFACT EDI? 
That makes the short form EDIFACT EDI Connector.
Since this is a Level-1 topic, I suggest reintroducing the whole shebang:
```suggestion
Using the Write operation in the Anypoint Connector for EDIFACT EDI (EDIFACT EDI Connector), you can generate control numbers on the generated EDIFACT payload by:
```

Reply:
Question: you use Write to _generate_ control numbers by..._managing_ or _using_? 
This doesn't sound right. To me, _generate_ is like create or obtain, rather than manage or use.
Doublecheck?
(Also, see if you can avoid using "generate" twice in the final clause.)